test/support/utils.js

```diff
@@ -26,7 +27,20 @@ function createError (message, props) {
   return err
 }
 
-function createServer (err, opts) {
+function createHTTPServer (err, opts) {
+  return http.createServer(function (req, res) {
+    var done = finalhandler(req, res, opts)
+
+    if (typeof err === 'function') {
+      err(req, res, done)
+      return
+    }
+
+    done(err)
+  })
+}
+
+function createHTTP2Server (err, opts) {
   return http.createServer(function (req, res) {
```

Comment:
Am I missing something here? Is this supposed to be using `http2`? Sorry if I am being silly here, barely started my morning coffee.